### DISCUSSION TITLE
是否需要 “安装整合包时保留原始整合包” 功能？

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#1480] [讨论]
![image](https://user-images.githubusercontent.com/40348278/226871433-a7e74dc0-26dc-4c4f-a888-eb712e9736ad.png)

该设置可能只对很少一部分玩家有用，绝大多数玩家应当不需要保留安装包（就像装了 APP 就应该清理掉一样）。
如果它的作用面太小，可能这是一个冗余的设置，所以搞个投票。

---

如果你 **开启了这个选项并且需要它** ，在下方选择 **上拇指** 。
如果你 **没有开启，或者不需要它** ，在下方选择 **下拇指** 。

**Comment by RawDiamondMC** [comment]
现在这个设置似乎是默认开启的，或许可以默认关闭？

**Comment by hejiehao** [comment]
如果真的需要保留，我认为仅需在下载时点击另存为按钮

**Comment by MoYuan-CN** [comment]
如果是投票的话,建议下
用 Github discussion 的 poll 类型,可直接创建投票选择栏

**Reply by Chiloven945** [reply]
但是显然这种方法仍然是有效的

**Reply by MoYuan-CN** [reply]
那当然 
~~但是过于简陋了 :-)~~

**Reply by Chiloven945** [reply]
需求是达到能显示大家意向的效果，不是为了好看或者是其他什么的

**Reply by iPodtouch6557** [reply]
> 需求是达到能显示大家意向的效果，不是为了好看或者是其他什么的

~是很简陋~但是可以用了，能达到效果就可以了
~mojang就这个宗旨，能用就行~

**Comment by iPodtouch6557** [comment]
我觉得可以在开了时把他们找个地集中打包

**Comment by WorldHim** [comment]
~~可以不用，但不能没有~~

**Reply by MoYuan-CN** [reply]
赞同

**Comment by LTCatt** [comment]
考虑投票结果，还是就保持这样不变吧 =。=
